tendrl/gluster_integration/sds_sync/vol_utilization.py

```diff
@@ -26,10 +26,14 @@ def showVolumeUtilization(vnames):
         if volume != "":
             try:
                 data = gfapi.getVolumeStatvfs(volume)
-            except gfapi.GlusterLibgfapiException:
-                sys.stderr.write("CRITICAL: Failed to get the "
-                                 "Volume Utilization Data\n")
-                sys.exit(-1)
+            except gfapi.GlusterLibgfapiException as e:
+                if 'error' not in ret_dict.keys():
```

Comment:
Declare ret_dict above at line#24 as `ret_dict = {"error": {}, "success": {}}`.
So you wont have to check if the key exists or not and creation of it.

Reply:
initially, he declared like this, I told him to change because in utilization.py file he does need to worry about the error.he can proceed with success. I feel it is better to create error only any problem issue raise.

Reply:
Ack. Fine.
If verified with both happy path and with problematic volumes, good to go.